cpp/daal/src/algorithms/dtrees/forest/classification/df_classification_predict_dense_default_batch_impl.i

```diff
@@ -278,7 +278,25 @@ void PredictClassificationTask<algorithmFPType, cpu>::predictByTrees(const size_
             PRAGMA_VECTOR_ALWAYS
             for (size_t i = 0; i < _nClasses; ++i)
             {
-                resPtr[i] += probas[idx * _nClasses + i] * inverseTreesCount;
+                resPtr[i] += probas[idx * _nClasses + i];
+            }
+
+            if (iTree + 1 == nTreesTotal)
```

Comment:
In case `iLastTree != nTreesTotatl` `resPtr` is not normalized. Is it expected?
it's a problem in case `resPtr` is not common for all trees blocks.

Reply:
There is no threading for trees in prediction, even in predictByBlocksOfTrees(). In this case an information from tree with last number be taken into account in the last, and normalization for all classes will be done in the end.